package.json

```diff
@@ -111,6 +112,7 @@
     "lint-staged": "^10.1.3",
     "nyc": "^15.0.0",
     "prettier": "2.0.2",
+    "puppeteer": "^3.0.4",
```

Comment:
This actually isn't used directly by our code (only indirectly via pageres), but I was worried about pageres using a pretty old version.